src/ledger.cc

```diff
@@ -168,23 +168,34 @@ void Ledger::process() {
           ++threshold_itr) {
         double pct =
           (*threshold_itr < 0) ? -(*threshold_itr) : (*threshold_itr);
+         int num_in_threshold = 0;
```

Comment:
It's used as an array pointer so it has to be an int to hit an index in the array.